### DISCUSSION TITLE
How to limit journey mode research

### DISCUSSION THREAD

**Discussion by ghost** [Pryaxis/TShock#2068] [Q&A] [answered]
is there any way to limit what people can research through journey mode (built into TShock or a plugin), or is it even possible. what i am wanting to do is not completely disable it but only allow certain items to be researched eg: allowing research for building blocks, but not mats, weps and armor. i don't exactly mean just putting stuff in categories because that would not work for everyone, but just able to disable certain item id's from being researched. (sorry for wall of text just want to be clear)

**Comment by hakusaro** [comment, marked as answer]
If you have SSC enabled, it’s possible for plugins to restrict what is or isn’t researchable. You can kind of hack the game a bit to restrict things without SSC. I don’t think anyone has built anything for this right now, though.

**Reply by ghost** [reply]
I planed on using SSC so that is good, guess i need to figure out how to code a plugin then, should not be too difficult considering i already know some coding, just takes getting used to a new language, will make sure to share it if i do make it. Thank you!
